commit 5b45c30a00ecbd5181e5679b13ca059bcd761ead
Author: Luca Boccassi <luca.boccassi@gmail.com>
Date:   Thu Jun 22 13:04:59 2017 +0100

    ethdev: add missing symbol in map
    
    The function rte_eth_tx_done_cleanup() was missing in the map file
    so it cannot be used by applications linking to shared libraries.
    pktgen uses it since version 3.2.0.
    
    Fixes: 44a718c457b5 ("ethdev: add API to free consumed buffers in Tx ring")
    Cc: stable@dpdk.org
    
    Signed-off-by: Luca Boccassi <luca.boccassi@gmail.com>

diff --git a/lib/librte_ether/rte_ether_version.map b/lib/librte_ether/rte_ether_version.map
index 805e6de80..e0881f09d 100644
--- a/lib/librte_ether/rte_ether_version.map
+++ b/lib/librte_ether/rte_ether_version.map
@@ -142,6 +142,7 @@ DPDK_17.05 {
 
 	rte_eth_dev_attach_secondary;
 	rte_eth_find_next;
+	rte_eth_tx_done_cleanup;
 	rte_eth_xstats_get_by_id;
 	rte_eth_xstats_get_id_by_name;
 	rte_eth_xstats_get_names_by_id;
